CustomEnvironment.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as path from 'path';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const NodeEnvironment = require('jest-environment-node').TestEnvironment;
+
+/**
+ * Extracts the test name from an absolute path received by the context
+ * @param {string} filePath Absolute path
+ * @returns {string} Test filename without directories, test suffixes or extensions
+ * @example
+ * const name = getTestName('/home/user/code/address-info.test.js')
+ * assert(name == 'address-info')
```

Comment:
[not-blocking] This assert has a function on the docstring or it is just an illustration?

Reply:
It's just an illustration to indicate that the contents of `name` should be `address-info`.